docs/guides/api-offchain.mdx

```diff
@@ -0,0 +1,310 @@
+  "value": 0.031 // Price in Ether (Decimals Price)
+}
+```
+
+A successful response will have the following message:
+
+```json
+{
+    "message": "successfully simulated contract call"
+}
+```
+This means that the order is still valid and can be filled.
+
+---
+
+## Submitting
+
+Now that we have retrieved the calldata for the order and confirmed that it is valid, we can submit a transaction to the blockchain to fill the order.
+```js
+// Uses an ethers.js signer instance: 
+// You can get the signer from a wallet using web3modal/rainbowkit etc.
+// See: https://docs.ethers.io/v5/api/signer/
+
+import { ethers } from "ethers";
+const provider = new ethers.providers.Web3Provider(web3.currentProvider, 1);
+const signer = await provider.getSigner();
+
+const decimalPrice = "0.031"; // Price in ETH
+const weiPrice = ethers.utils.parseEther(decimalPrice);
+
+const tx = {
+  from: "YOUR_ADDRESS",
+  to: "0x00000000006c3852cbef3e08e8df289169ede581", // Contract that fills the order e.g. Seaport
+  value: weiPrice, // Price in Wei
+  data: "0xfb0f3ee10000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000063c8117529600000000000000000000000000025659eefd7cae21566a257420bbcb0c8317ef94f000000000000000000000000004c00500000ad104d7dbd00e3ae0a5c00560c00000000000000000000000000ca21d4228cdcc68d4e23807e5e370c07577dd1520000000000000000000000000000000000000000000000000000000000007c2e000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000062e930e10000000000000000000000000000000000000000000000000000000063120f61000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001538739839173460000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000024000000000000000000000000000000000000000000000000000000000000002e000000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000002c0dbbfb370000000000000000000000000008de9c5a032463c561423387a9648c5c7bcc5bc9000000000000000000000000000000000000000000000000000079968bf2cb000000000000000000000000000d1d1d4e36117ab794ec5d4c78cbd3a8904e691d0000000000000000000000000000000000000000000000000000000000000004169d9d167a41f282ee33134ce0fa3041b458904ad1c85255f712dd6dd10f1ad8c65dd4534f4a5d1f5ee936224bc50fea910864530a3f06e534d89c6c2a1fdb5021b00000000000000000000000000000000000000000000000000000000000000" ,
+}
+
+const result = await signer.sendTransaction(tx)
+```
```

Comment:
This looks awesome, only nit is imo we should add "end to end", fetch a bunch of orders from the api, select one, and submit the order instead of having hte data copy pasted over